### ISSUE TITLE
java.lang.NoClassDefFoundError: com/jgoodies/common/base/Strings

### ISSUE THREAD

**Issue by haimivan** [zielu/SVNToolBox#30]
Hi,

I get an error. I am not sure whether I can describe how this happened.

Bye!

> Ubuntu 14.04, 
> PyCharm 2019.1.1 (Professional Edition)
> Build #PY-191.6605.12, built on April 3, 2019
> JRE: 11.0.2+9-b159.34 amd64
> JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
> Linux 4.4.0-146-generic
> 

```
java.lang.NoClassDefFoundError: com/jgoodies/common/base/Strings
	at zielu.svntoolbox.lockinfo.SvnLockOwnerComponent.getOwner(SvnLockOwnerComponent.java:86)
	at zielu.svntoolbox.ui.actions.ShowLockInfoTask.run(ShowLockInfoTask.java:94)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:731)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:586)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:86)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:403)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:311)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassNotFoundException: com.jgoodies.common.base.Strings PluginClassLoader[zielu.svntoolbox, 191.0.0] com.intellij.ide.plugins.cl.PluginClassLoader@3c82a996
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:51)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	... 15 more
```
```

**Comment by zielu** [comment]
Fixed in 191.0.1

**Comment by haimivan** [comment]
cool! :-)
